docs/flow/reference/components/prometheus.relabel.md

```diff
@@ -0,0 +1,181 @@
+    action        = "keep"
+    regex         = "backend"
+  }
+
+  metric_relabel_config {
+    action = "labeldrop"
+    regex  = "instance"
+  }
+}
+```
+
+Let's see how the previous instance of the `prometheus.relabel` component would
+act on some metrics.
+
+```
+metric_a{__address__ = "localhost", instance = "development", app = "frontend"} 10
+metric_a{__address__ = "localhost", instance = "development", app = "backend"}	2
+metric_a{__address__ = "cluster_a", instance = "production",  app = "frontend"} 7
+metric_a{__address__ = "cluster_a", instance = "production",  app = "backend"}	9
+metric_a{__address__ = "cluster_b", instance = "production",  app = "database"}	4
+```
+
+After applying the first `metric_relabel_config` block, the `replace` action
+would populate a new label named `host` by the concatenating the two
+`__address__` and `instance` labels separated by a slash `/`.
+
+```
+metric_a{host = "localhost/development", __address__ = "localhost", instance = "development", app = "frontend"} 10
+metric_a{host = "localhost/development", __address__ = "localhost", instance = "development", app = "backend"}	2
+metric_a{host = "cluster_a/production",  __address__ = "cluster_a", instance = "production",  app = "frontend"} 7
+metric_a{host = "cluster_a/production",  __address__ = "cluster_a", instance = "production",  app = "backend"}	9
+metric_a{host = "cluster_b/production",  __address__ = "cluster_a", instance = "production",  app = "database"}	4
+```
+
+After applying the second relabeling step, the `keep` action would only keep
+the metrics whose `app` label match `regex`, dropping everything else, so the
+list of metrics would be trimmed down to 
+
+```
+metric_a{host = "localhost/development", __address__ = "localhost", instance = "development", app = "backend"}	2
+metric_a{host = "cluster_a/production",  __address__ = "cluster_a", instance = "production",  app = "backend"}	9
+```
+
+The third and final relabeling step which uses the `labeldrop` action would
+remove the `instance` label from the set of labels
+
+```
+metric_a{host = "localhost/development", __address__ = "localhost", app = "backend"}	2
+metric_a{host = "cluster_a/production",  __address__ = "cluster_a", app = "backend"}	9
+```
+
+Finally, the `__address__` label, as it starts with a double underscore and is
+reserved as an 'internal' label and would be implicitly removed.
+
+After filtering down the samples passed through the exported receiver, these
+two final metrics would then be propagated to each receiver passed in the
+`forward_to` argument.
+```
+metric_a{host = "localhost/development", app = "backend"}	2
+metric_a{host = "cluster_a/production",  app = "backend"}	9
+```
+
+## Arguments
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+forward_to | list(receiver) | Where the metrics should be forwarded to after relabeling takes place | | **yes**
```

Comment:
Nit: I'd like for us to wrap names and types in backticks so forward_to appears as `forward_to`. 

Ditto for elsewhere in this file and when we describe the list of supported relabel actions.